queries/customer-dashboard.gql

```diff
@@ -0,0 +1,43 @@
+query CustomerDashboard {
+  billing(where: { customerId: { _eq: "7" } }) {
```

Comment:
Any reason why we're hard coding this as 7 when `CustomerBehaviorAnalysis` above uses a variable? I'm easy with either but consistency across both would be best.